scripts/all.sh

```diff
@@ -2,7 +2,7 @@
 
 set -eu
 
-HERE="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+HERE="$( cd -- "$(dirname "$0")" >/dev/null 2>&1 ; pwd -P )"
```

Comment:
I'm a bit of a bash noob, would you mind explaining what this change does for me?

Reply:
Basically the same, getting the script dir, `readlink -f` doesn't work on MacOs.